rs/nns/BUILD.bazel

```diff
@@ -0,0 +1 @@
+# This file intentionally left blank.
```

Comment:
Is this empty BUILD.bazel needed? Same question for `rs/sns/BUILD.bazel`.

Reply:
It's not STRICTLY necessary, but if I remove them, I have to change from

```
load("//rs/sns:sns.bzl", ...)
```

to

```
load("//rs:sns/sns.bzl", ...)
```

I would want to avoid the latter, because in general, there are no slashes after the semicolon, so it might throw some ppl off.

Reply:
Ok sounds good.